users/models.py

```diff
@@ -8,3 +10,34 @@ class Person(models.Model):
 
     class Meta:
         abstract = True
+
+
+class Club(models.Model):
```

Comment:
ik zou de arguments of allemaal direct een enter geven en dan indent van 4, of zoveel mogelijk inline en alleen de te lange regels afbreken en indent gelijk houden aan eerste argument. nu rommelt het een beetje. snap je wat ik bedoel?

Vergeet je niet de migratie?